packages/app/src/components/nominationInfo/index.js

```diff
@@ -1,20 +1,33 @@
 import React, { useState, useContext, useEffect } from 'react';
 import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
 import ApplicationDetail from "./applicationDetail";
+import HealthProviderDetail from "./healthProviderDetail";
 import styles from "./styles.module.css";
 
 
+
+
+
+
 function NominationInfo() {
   const [activeNomination, setActiveNomination] = useContext(
     ActiveNominationContext
   );
 
+  
+
+
+
+  const hospitalAddress = activeNomination.hospitalCity+', '+activeNomination.hospitalState+', '+activeNomination.hospitalZipCode
```

Comment:
you can [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) the `activeNomination` object to reduce redundancy. and then use [javascript templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to combine the strings.

I think it looks cleaner using a template opposed to string concatenation. 

it would look something like:
```javascript 
const { hospitalCity, ... } = activeNomination;
const hospitalAddress = `${hospitalCity}, the rest of your string here`;
```

Reply:
Okay, gottya.  I just updated the branch to include these changes.